src/transformers/models/encodec/modeling_encodec.py

```diff
@@ -643,16 +656,24 @@ def encode(
         for offset in range(0, input_length - step, stride):
             mask = padding_mask[..., offset : offset + chunk_length].bool()
             frame = input_values[:, :, offset : offset + chunk_length]
-            encoded_frame, scale = self._encode_frame(frame, bandwidth, mask)
+            encoded_frame, scale, quantization_steps = self._encode_frame(frame, bandwidth, mask)
             encoded_frames.append(encoded_frame)
+            quantization_steps_frames.append(quantization_steps)
             scales.append(scale)
 
         encoded_frames = torch.stack(encoded_frames)
 
+        commitment_loss = torch.tensor([0.0], device=input_values.device, requires_grad=self.training)
```

Comment:
Ideally `commitment_loss` is `None` if we're **not** training, and a torch tensor otherwise